package.json

```diff
@@ -6,7 +6,8 @@
     "build": "greenwood build",
     "develop": "greenwood develop",
     "start": "npm run develop",
-    "serve": "rimraf ./public && npm run build && ws -d ./public"
+    "serve": "rimraf ./public && npm run build && ws -d ./public",
+    "lint:js": "eslint \"*.js\" \"./src/**/**/*.js\""
```

Comment:
More of a favor, but was thinking maybe it would make sense to revert / save this for a standalone PR, but in that new PR, we could then also include **stylelint** and **ls-lint**?  From there, we could then wire that all up into our GitHub actions flow.

Thoughts?